password_store/lib/password_store/service/impl.ex

```diff
@@ -0,0 +1,42 @@
+defmodule PasswordStore.Service.Impl do
+
```

Comment:
A list of entries seems a little strange: could could have used a map, keyed on username.